### ISSUE TITLE
Doesn't work with newest chrome - 49.0.2623.75 (64-bit)

### ISSUE THREAD

**Issue by selrond** [AliasIO/Wappalyzer#1115]
Doesn't show used extensions on any page, though appears installed and working in `chrome://extensions/`

**Comment by klonos** [comment]
Works fine for me on both x86 and x64. It's just that sometimes the icon is disabled but i comes back after a page refresh. Also do note that it won't work on the new tab page (the one you get by default where there is no URL in the address bar).

**Comment by selrond** [comment]
Well - found out it's working, but only after I press its button in the extensions... Shouldn't it be in url bar as well? Because it's completely missing there. (reloading does not help)

**Comment by klonos** [comment]
> ...Shouldn't it be in url bar as well? Because it's completely missing there.

#1110 (#1112 too, but was closed as duplicate of the other issue).

**Comment by selrond** [comment]
Exactly thanks

**Comment by klonos** [comment]
No worries.
